pyproject.toml

```diff
@@ -63,8 +63,8 @@ dependencies = [
     "pyspark[connect]==3.5.1",
     # We do not specify the extra dependencies for Ibis since we install PySpark explicitly.
     "ibis-framework~=9.4",
-    "pyarrow-hotfix>=0.4,<1",  # (required by Ibis PySpark)
-    "rich>=12.4.4,<14",  # (required by Ibis PySpark)
+    "pyarrow-hotfix>=0.4,<1", # (required by Ibis PySpark)
+    "rich>=12.4.4,<14", # (required by Ibis PySpark)
```

Comment:
Let's revert this. It's common to have two spaces before comment. For example, the Python formatter actually enforces this in `.py` files.

Reply:
Never mind. It's OK to keep IDE formatting on the file, until we add CI formatter for TOML files in the future.